Project.toml

```diff
@@ -31,6 +36,7 @@ PencilFFTs = "4a48f351-57a6-4416-9ec4-c37015456aae"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
@jagoosw we don't want these packages in Oceananigans Project.toml. Can you remove them?

Reply:
same for Measures, GLMakie, InteractiveErrors, CairoMakie, BenchmarkTools

Reply:
Apologies didn't mean to add these